### PR TITLE
KTOR-6613 Add logging to the client application example and improve docs

### PR DESCRIPTION
**Pull request by vnikolova**
[KTOR-6613](https://youtrack.jetbrains.com/issue/KTOR-6613) 

Running the application produces the following warning:

> SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
> SLF4J: Defaulting to no-operation (NOP) logger implementation
> SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.

Preview on staging:
- [Create a client application](https://ktor.shared.aws.intellij.net/docs/client-create-new-application.html#make-request) -updated the procedure to include the SLF4J error and added a new section for enabling logging to avoid the error
- [Adding client dependencies](https://ktor.shared.aws.intellij.net/docs/client-dependencies.html) - added a logging dependency section
- [Logging](https://ktor.shared.aws.intellij.net/docs/client-logging.html) - expanded the explanation

### CODE REVIEW

File: topics/client-create-new-application.topic [hunk 1]
```diff
@@ -244,9 +244,31 @@
                 <code-block
                         lang="kotlin"
                         src="snippets/tutorial-client-get-started/src/main/kotlin/Main.kt"/>
+            </step>
+            <step>
                 <p>
-                    Proceed to the next section to learn how to run your application.
+                    To see logs correctly, you need to add a logger dependency. In the
```
Comment (Stexxe) [inline topics/client-create-new-application.topic:250]: Should we explain how the logging works and about SLF4J?

**Review by Stexxe — APPROVED** [review]

File: topics/client-create-new-application.topic (hunk 1 above)
Comment (vnikolova) [reply topics/client-create-new-application.topic]: I've added more information in the relevant topics. (preview links in description) 
Can you please verify?

File: topics/client-logging.md [hunk 2]
```diff
@@ -10,30 +10,49 @@
...
-This plugin provides different logger types for different platforms:
+This plugin provides different logger types for different platforms.
+
+## JVM
+
+<snippet id="jvm-logging">
+  <p>
+    On <a href="client-engines.md" anchor="jvm">JVM</a>, Ktor uses the Simple Logging Facade for Java
+    (<a href="http://www.slf4j.org/">SLF4J</a>) as an
+    abstraction layer for logging. This allows you to bind logging with a framework of your choice according to your needs
```
Comment (Stexxe) [inline topics/client-logging.md:27]: What does it mean to "bind logging with a framework of your choice"?

Comment (vnikolova) [reply topics/client-logging.md:27]: Yeah "logging" is too vague in this sentence. Perhaps this adds clarity: "SLF4J decouples the logging API from the underlying logging implementation, allowing you to integrate the logging framework that best suits your application's requirements." ?

File: v.list [hunk 3]
```diff
@@ -8,7 +8,7 @@
     <var name="kotlin_version" value="1.9.10"/>
     <var name="coroutines_version" value="1.6.4"/>
     <var name="kotlin_css_version" value="1.0.0-pre.625"/>
-    <var name="logback_version" value="1.2.11"/>
+    <var name="logback_version" value="1.5.3"/>
```
Comment (e5l) [inline v.list:11]: Please make sure this version works with JDK 11

**Review by e5l — APPROVED** [review]

**Pull request merged**